dependencies.tsv

```diff
@@ -1,6 +1,8 @@
+github.com/frankban/quicktest	git	fe41ed117766719e2d387ef4dfd9b12846de9da9	2017-09-21T13:10:42Z
```

Comment:
It'd be nice if you only needed one of gocheck and quicktest

Reply:
Will fix in a subsequent PR.